Upcoming release/c8097ed3-15e7-4941-8eb1-7bed1fadef9d.md

```diff
@@ -80,14 +81,16 @@ To search by products, send the request:
                         "name_asc",
                         "name_desc",
                         "price_asc",
-                        "price_desc"
+                        "price_desc",
+                        "popularity"
```

Comment:
desc? Or asc?

Reply:
I've seen that it uses DESC direction. But property for sorting is called 'popularity'.